packages/components/src/templates/next/components/complex/Hero/HeroLargeImage/ScrollForMoreButton.tsx

```diff
@@ -0,0 +1,31 @@
+interface ScrollForMoreButtonProps {
+  isFixed: boolean
+}
+
+export const ScrollForMoreButton = ({ isFixed }: ScrollForMoreButtonProps) => {
+  return (
+    <div
+      className={`${isFixed ? "fixed" : "absolute"} bottom-16 left-1/2 -translate-x-1/2 animate-slide-up-fade-in rounded-full`}
```

Comment:
[sense-check] when i had a conditional like this in my code, it didn't _quite_ work wrt styling. can we double check that it's workign here (also, if you know why it didn't work last time, it'd be much appreciated! i swapped over to using `tv` to get it to work)

Reply:
was that a build-specific issue? because checked that this is working on storybook. nevertheless updated it to use tv - i think its cleaner too